files/rsyslog/65-viaq-formatting.conf

```diff
@@ -246,14 +254,12 @@ if strlen($!level) > 0 then {
     unset $.lclevel;
     unset $.normlevel;
 } else {
-    if $!stream == "stdout" then {
+    if $!message startswith "I" then {
```

Comment:
No, please don't add this yet.  I think we need a separate commit to fix https://bugzilla.redhat.com/show_bug.cgi?id=1726639 `fluentd misbehaving and marking an info log wrong level to 'err'` - we will need to add more parsing, and we will need to coordinate the fix with fluentd.